modules/images/webp-uploads/image-edit.php

```diff
@@ -120,7 +120,8 @@ function ( $metadata, $post_meta_id ) use ( $post_id, $file_path, $mime_type, $e
 
 			$old_metadata = wp_get_attachment_metadata( $post_id );
 			$resize_sizes = array();
-			$target       = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all';
+			// PHPCS ignore reason: A nonce check is not necessary here as this logic directly ties in with WordPress core logic which already has one.
```

Comment:
We are modifying the behavior of what gets saved to the database based on the value of this `$_REQUEST` property, so it would be best if we could identify where the nonce check is being handled.